English.pm uses Errno.pm which checks that it is on the target platform

--- XML-Parser-2.36/Expat/Makefile.PL.orig	2009-02-27 14:57:00.000000000 +0100
+++ XML-Parser-2.36/Expat/Makefile.PL	2009-02-27 14:57:17.000000000 +0100
@@ -1,6 +1,6 @@
 use ExtUtils::MakeMaker;
 use Config;
-use English;
+# use English;
 
 my $libs = "-lexpat";
 @extras = ();
